src/pages/Embeds/Plio.vue

```diff
@@ -413,6 +413,9 @@ export default {
      * whether the scorecard is enabled or not
      */
     isScorecardEnabled() {
+      if (this.areAllQuestionsSurvey()) {
+        return;
+      }
       return this.items != undefined && this.hasAnyItems;
```

Comment:
Would recommend keeping `areAllQuestionsSurvey` as a computed variables so that it is not recomputed everytime. Currently, everytime that you call `areAllQuestionsSurvey()`, it is being recomputed even though the value is going to stay the same. Plus, you can simply write:

```suggestion
      return this.items != undefined && this.hasAnyItems && !this.areAllQuestionsSurvey;
```